internal/test/wmcb/main_test.go

```diff
@@ -17,7 +17,9 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	err := framework.Setup(vmCount)
+	// For development purposes, you can pass in credentials of already existing VMs
+	// credentials := []*types.Credentials{types.NewCredentials("instanceID", "ip", "password")}
+	err := framework.Setup(vmCount, nil, false)
```

Comment:
Expose this to the enduser please via cli flag. It'd be ideal to have `hack/ci-` to have this flag to skipVM creation. You can get the flag values and instantiante the framework here https://github.com/openshift/windows-machine-config-operator/pull/122/files#diff-f742eafb78aace92926680fb9193bfe9R13

Reply:
What exactly do you want to expose via the CLI flag? The list of VM credentials?

Reply:
Ideally both the `skipVM` creation flag and credentials but I see that in the current approach we're not persisting credentials to a file from which we can read, so we can try passing VM creds list and see how it goes.

Reply:
@ravisantoshgudimetla I have added support for CLI flags. PTAL.